lambda/update/certificate_provider_sign.go

```diff
@@ -48,6 +62,12 @@ func validateCertificateProviderSign(changes []shared.Change) (CertificateProvid
 		Field("/certificateProvider/contactLanguagePreference", &data.ContactLanguagePreference, parse.Validate(func() []shared.FieldError {
 			return validate.IsValid("", data.ContactLanguagePreference)
 		})).
+		Field("/certificateProvider/email", &data.Email, parse.Validate(func() []shared.FieldError {
```

Comment:
I spoke with Abbi about this to check if there was a need for the email address of a paper -> online CP before signing and Abbi couldn't think of a scenario where a case worker/OPG would use it.

I guess it may be useful to know the correct email sooner for the scenario of online CPs that have a different email address than provided by the donor?

Reply:
that's cool, I just wanted to confirm that we weren't doing what the story says 🙃